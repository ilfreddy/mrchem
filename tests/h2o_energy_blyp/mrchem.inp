# vim:syntax=sh:

rel_prec = 1.0e-3                 # Overall relative precision

MRA {
  order = 6                       # Polynomial order
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
$coords
O       0.0000  0.0000  -0.1250
H      -1.4375  0.0000   1.0250
H       1.4375  0.0000   1.0250
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
}

DFT {
$functionals
BLYP
$end
}

Properties {
  scf_energy = true               # Compute ground state energy
  dipole_moment = true            # Compute electrostatic dipole moment
}

SCF {
  run = false
  canonical = false               # Use canonical of localized orbitals
  initial_guess = GTO             # Type of initial guess: none, mw, gto
}

