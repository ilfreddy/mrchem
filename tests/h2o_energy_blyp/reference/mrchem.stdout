


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. 839f67b)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 735162b

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 6
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -32.000000  -32.000000  -32.000000 ]
 upper bounds     = [   32.000000   32.000000   32.000000 ]
 total length     = [   64.000000   64.000000   64.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     O           0.00000e+00   0.00000e+00  -1.25000e-01
  2     H          -1.43750e+00   0.00000e+00   1.02500e+00
  3     H           1.43750e+00   0.00000e+00   1.02500e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   3.68310e-03
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     O            8.00000e+00   1.00000e-03   5.10129e-04
  2     H            1.00000e+00   1.00000e-03   1.63241e-02
  3     H            1.00000e+00   1.00000e-03   1.63241e-02
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    p    2                       1.000000203953468e+00
    1    p    2                       1.000000065236651e+00
    2    p    2                       9.999999991988902e-01
    3    p    2                       1.000000008869737e+00
    4    p    2                       9.999999986945859e-01
------------------------------------------------------------
                 Wall time: 3.80590e+00 sec 
============================================================


============================================================
 OrbitalVector:   5 orbitals     5 occupied    10 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       1.000000203953e+00    p   2    1.00000e+00
   1     0       1.000000065237e+00    p   2    1.00000e+00
   2     0       9.999999991989e-01    p   2    1.00000e+00
   3     0       1.000000008870e+00    p   2    1.00000e+00
   4     0       9.999999986946e-01    p   2    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                    1384      8.15220e-01
 Coulomb density                       520      5.89911e-01
 Coulomb potential                     328      2.94648e+00
 XC total density                      520      5.96354e-01
 XC evaluate xcfun                    2600      3.30720e+00
 XC energy                             520      4.35020e-05
 XC potential                          520      9.87510e-02
 XC grid size                          520          1251264
 XC grid change                       -512         -1229312
------------------------------------------------------------
                 Wall time: 8.42214e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         1.241902587000000e+00
 Potential part                       2.344024419000000e+00
------------------------------------------------------------
                 Wall time: 3.58599e+00 sec 
============================================================


============================================================
                    Localizing orbitals
------------------------------------------------------------
 Computing position matrices                    2.37270e+00
 Computing Foster-Boys matrix                   2.53498e-02
 Converged after iteration                               41
 Rotating orbitals                              1.77163e-01
------------------------------------------------------------
                 Wall time: 2.57529e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                    1384      7.76732e-01
 Coulomb density                       552      6.46175e-01
 Coulomb potential                     328      2.95903e+00
 XC total density                      552      6.27038e-01
 XC evaluate xcfun                    2760      3.48918e+00
 XC energy                             552      4.21880e-05
 XC potential                          552      1.02417e-01
 XC grid size                          552          1328096
 XC grid change                       -544         -1306144
------------------------------------------------------------
                 Wall time: 8.67255e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 2.70262e+00 sec 
============================================================


============================================================
                 Calculating dipole moment
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 2.48924e+00 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-1.86630e+01  4.81610e-01  4.81610e-01 -9.18467e-01  9.18467e-01
 4.81611e-01 -5.98871e-01 -1.51812e-01  1.58107e-01 -1.58107e-01
 4.81611e-01 -1.51812e-01 -5.98872e-01  1.58107e-01 -1.58107e-01
-9.18469e-01  1.58107e-01  1.58107e-01 -3.94212e-01  1.71620e-01
 9.18469e-01 -1.58107e-01 -1.58107e-01  1.71620e-01 -3.94212e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    p    2                      -1.878310621020190e+01
    1    p    2                      -8.928172744224612e-01
    2    p    2                      -4.470593336320008e-01
    3    p    2                      -3.036190323441361e-01
    4    p    2                      -2.225912897045666e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     O           0.00000e+00   0.00000e+00  -1.25000e-01
  2     H          -1.43750e+00   0.00000e+00   1.02500e+00
  3     H           1.43750e+00   0.00000e+00   1.02500e+00
------------------------------------------------------------
 Center of mass:    0.00000e+00   0.00000e+00   3.68310e-03
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -4.129838628061009e+01
 Kinetic energy:                      7.584717193641946e+01
 E-N energy:                         -1.987273124965973e+02
 Coulomb energy:                      4.676221778048411e+01
 Exchange energy:                     0.000000000000000e+00
 X-C energy:                         -9.318930593346984e+00
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -8.543685337304075e+01
 Nuclear energy:                      9.039235444235468e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.639761792880527e+01
                    (kJ/mol)         -2.005819168409834e+05
                    (kcal/mol)       -4.794022865936947e+04
                    (eV)             -2.078884998363504e+03
                                                            
============================================================
                                                            

                                                            
============================================================
                         Dipole moment                      
------------------------------------------------------------
                                                            
 Length of vector:   (au)                  7.7928754277e-01
                     (Debye)               1.9807511747e+00
                                                            
------------------------------------------------------------
                                                            
   7.5414125273e-07   -9.0984454942e-12    7.7928754277e-01
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 3.244713e+01                ***
***                                                      ***
************************************************************


